### ISSUE TITLE
Plugin 'mobi.hsz.idea.gitignore' failed to initialize and will be disabled.  Please restart PhpStorm.

### ISSUE THREAD

**Issue by Didldu-Florian** [hsz/idea-gitignore#328]
Plugin 'mobi.hsz.idea.gitignore' failed to initialize and will be disabled.  Please restart PhpStorm.

Not sure why, we did installed hsz/idea-gitignore plugin at a new desktop PC. Java last release is also installed. Attached some System Infos:

PhpStorm 2016.3.2
Build #PS-163.10504.2, built on December 20, 2016
JRE: 1.8.0_60-b27 amd64
JVM: Java HotSpot(TM) 64-Bit Server VM by Oracle Corporation

After installing Plugin and restarting, PhpStorm did disable Plugin and following error did display.

![error2](https://cloud.githubusercontent.com/assets/12803406/23170414/5cbfc688-f84f-11e6-94e8-42e047067bd4.jpg)

**Comment by Nemmo** [comment]
Exactly the same error on the mac and windows

**Comment by fatcats** [comment]
Same error

**Comment by fatcats** [comment]
![qq 20170222015644](https://cloud.githubusercontent.com/assets/12790403/23177577/3da7219e-f8a2-11e6-9ae4-a2ea887df187.jpg)

**Comment by Grafe89** [comment]
Same here

**Comment by kraluk** [comment]
Same here. Even if I've reactivated it after a crush.

```
Plugin 'mobi.hsz.idea.gitignore' failed to initialize and will be disabled.  Please restart IntelliJ IDEA.

java.lang.NullPointerException
    at mobi.hsz.idea.gitignore.IgnoreApplicationComponent.initComponent(IgnoreApplicationComponent.java:59)
    at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:501)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:125)
    at com.intellij.openapi.application.impl.ApplicationImpl.a(ApplicationImpl.java:444)
    at com.intellij.openapi.progress.impl.CoreProgressManager$3.run(CoreProgressManager.java:179)
    at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
    at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
    at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:164)
    at com.intellij.openapi.application.impl.ApplicationImpl.createComponents(ApplicationImpl.java:451)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:109)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:403)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:389)
    at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:190)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at com.intellij.idea.MainImpl$1$1.a(MainImpl.java:68)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:709)
    at java.awt.EventQueue$3.run(EventQueue.java:703)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:363)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82) 
```

**Comment by Trak-X** [comment]
Same for my Android Studio with 1.7.5 from the repo. I downloaded manually the 1.7.4 and installed from disk and no issue here. It asks me to update, but i ignore it.

**Comment by plombardi89** [comment]
Same here

```
Plugin 'mobi.hsz.idea.gitignore' failed to initialize and will be disabled.  Please restart IntelliJ IDEA.

java.lang.NullPointerException
    at mobi.hsz.idea.gitignore.IgnoreApplicationComponent.initComponent(IgnoreApplicationComponent.java:59)
    at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:501)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:125)
    at com.intellij.openapi.application.impl.ApplicationImpl.a(ApplicationImpl.java:444)
    at com.intellij.openapi.progress.impl.CoreProgressManager$3.run(CoreProgressManager.java:179)
    at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
    at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
    at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:164)
    at com.intellij.openapi.application.impl.ApplicationImpl.createComponents(ApplicationImpl.java:451)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:109)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:403)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:389)
    at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:190)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at com.intellij.idea.MainImpl$1$1.a(MainImpl.java:68)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:709)
    at java.awt.EventQueue$3.run(EventQueue.java:703)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:363)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

IntelliJ Info

```
IntelliJ IDEA 2016.3.4
Build #IU-163.12024.16, built on January 31, 2017
Licensed to <REDACTED>
Subscription is active until March 17, 2017
JRE: 1.8.0_112-release-408-b6 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

**Comment by marcelopm** [comment]
:+1:

**Comment by pedroadame** [comment]
:+1:

**Comment by hsz** [comment]
Thank you for the report. It has been already reported and fixed with #315
